File: Modul_4/Bahasa Python/Prak405_2310817210013_MuhammadAufaFitrianda.py
n, kelipatan = map(int,input("Masukkan Nilai N dan Kelipatan Anda\t: ").split())

total = 0

for i in range(1, n + 1):

    baris = 0

    for j in range(i, 0, -1):
        
        baris += j * kelipatan
        if j == i:
            print(f"({j} * {kelipatan})", end="")
        else:
            print(f" + ({j} * {kelipatan})", end="")

    print(f" = {baris}")
    total += baris

print(total)

File: Tugas_Praktikum_Pemograman_1_Muhammad_Aufa_Fitrianda_2310817210013/Modul_4/Bahasa_C/Prak405_2310817210013_MuhammadAufaFitrianda.c
#include <stdio.h>

void main() {
    int n, kelipatan;
    int total = 0;

    printf("Masukan Nilai N Dan Kelipatan Anda\t: ");
    scanf("%d %d", &n, &kelipatan);

    for (int i = 1; i <= n; i++) {

        int baris = 0;

        for (int j = i; j > 0; j--) {

            baris += j * kelipatan;
            if (j == i) {
                printf("(%d * %d)", j, kelipatan);
            } 
            else {
                printf(" + (%d * %d)", j, kelipatan);
            }
            
        }

        printf(" = %d\n", baris);
        total += baris;
    }

    printf("%d\n", total);
}